site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -167,23 +167,29 @@ For improved security, all FusionAuth hosted login pages add an HTTP response he
 +
 Examples of valid authorized origin URIs:
 +
-* \https://example.com
-* com.myApp://example
-* com.myApp:/example
+ * \https://example.com
+ * com.myApp://example
+ * com.myApp:/example
```

Comment:
I believe we always allowed `*` for this value. Did we change this behavior now?

Reply:
Or was I just using this wrong?

Reply:
@mooreds are you talking about the use of a wildcard in the authorized URL fields? Or the formatting of this bulleted list?

I assumed that wildcards were not allowed because they were not processed prior to this feature being released. I'm going to correct that verbiage.

The leading space is required to prevent the text from the following paragraph being indented to be even with the text on list items.

Reply:
Sorry, was talking about using `*` for authorized origin URLS, previous to this release.

Reply:
Okay. I believe I've corrected that verbiage.